airbyte-integrations/connectors/source-simplesat/manifest.yaml

```diff
@@ -0,0 +1,399 @@
+version: 5.8.1
+
+type: DeclarativeSource
+
+description: |
+  Website : https://www.simplesat.io/
+  API Docs : https://documenter.getpostman.com/view/457268/SVfRt7WJ?version=latest
```

Comment:
Can you add a textual description instead of links?

Reply:
Sure!